File: 1000-1200 Python/CANDY123.py
T = int(input())

for i in range(T):
    temp = input().split(" ")
    A, B = int(temp[0]), int(temp[1])
    Limak, Bob, i, winner = 0, 0, 1, ""

    while winner == "":
        if i % 2 == 0:
            Bob += i

            if Bob > B:
                winner = "Limak"
                break
            else:
                i += 1

        else:
            Limak += i

            if Limak > A:
                winner = "Bob"
                break
            else:
                i += 1

    print(winner)

File: 1000-1200 C/CANDY123.c
    #include <stdio.h>
    #include <stdlib.h>
    int main(){
        int T, A, B;
        scanf("%d", &T);
        for (int i = 0; i < T; i++){
            int Limak = 0, Bob = 0, winner = 2, j = 1;
            scanf("%d %d", &A, &B);

            while (winner > 1){
                if (j % 2 == 0){
                    Bob += j;
                    if (Bob > B){
                        winner = 0;
                        break;
                    }
                    else{
                        ++j;
                    }
                }
                else{
                    Limak += j;
                    if (Limak > A){
                        winner = 1;
                        break;
                    }
                    else{
                        ++j;
                    }
                }
            }

            (winner == 0) ? printf("Limak\n") : printf("Bob\n");
        }   
        return 0;
    }
